sdk/recoveryservices-siterecovery/Microsoft.Azure.Management.RecoveryServices.SiteRecovery/src/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Recovery Services SiteRecovery Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Recovery Services SiteRecovery Resources.")]
 
-[assembly: AssemblyVersion("2.0.0.0")]
-[assembly: AssemblyFileVersion("2.0.3.0")]
+[assembly: AssemblyVersion("2.0.4.0")]
```

Comment:
```suggestion
[assembly: AssemblyVersion("2.0.0.0")]
```
Keep assembly version unchanged until a breaking change